src/libcore/ops.rs

```diff
@@ -95,6 +95,16 @@ use fmt;
 #[stable(feature = "rust1", since = "1.0.0")]
 pub trait Drop {
     /// A method called when the value goes out of scope.
+    ///
+    /// When this method has been called, `self` has not yet been deallocated.
```

Comment:
Actually, I'm not sure it even makes sense to use the word "deallocated," since a number of `Drop` impls don't have anything to with allocation. I think this description should probably just talk about it being invalid (or some other word) to access `self` after `drop` has been called.